mungegithub/mungers/submit-queue.go

```diff
@@ -896,6 +896,7 @@ const (
 	unmergeable             = "PR is unable to be automatically merged. Needs rebase."
 	undeterminedMergability = "Unable to determine is PR is mergeable. Will try again later."
 	noMerge                 = "Will not auto merge because " + doNotMergeLabel + " is present"
+	noReleaseNote           = "Will not auto merge because " + releaseNoteLabelNeeded + " is present. You need to set one of the release-note* labels."
```

Comment:
I think we should leave off the "You need to set.." bit as we'll have a comment on the PR, and a string that long won't show up properly in the Github UI.